specification/cognitiveservices/data-plane/AzureOpenAI/authoring/preview/2023-04-01-preview/images.json

```diff
@@ -0,0 +1,348 @@
+        "description": "Provide your Cognitive Services Azure OpenAI account key here."
+      }
+   },
+   "security": [
+      {
+        "Api-Key": []
+      }
+    ],
+   "parameters": {
+      "apiVersionQueryParameter": {
+        "in": "query",
+        "name": "api-version",
+        "description": "The requested API version.",
+        "required": true,
+        "type": "string"
+      }
+   },
+   "definitions": {
+      "Status": {
+         "description": "The status of the operation.",
+         "type": "string",
+         "enum": [
+            "NotStarted",
+            "Running",
+            "Failed",
+            "Canceled",
+            "Succeeded"
+         ]
+      },
+      "InnerError": {
+         "type": "object",
+         "description": "The inner error if a task in a batch failed.",
+         "properties": {
+            "message": {
+               "type": "string",
+               "description": "The error message.",
+               "example": "Operation failed as a result of our safety system."
+            },
+            "code": {
+               "type": "string",
+               "enum": [
+                  "ContentFilter",
+                  "Internal",
+                  "BlockedImages"
+               ],
+               "description": "The error code."
+            }
+         }
+      },
+      "Error": {
+         "type": "object",
+         "description": "The error if the request or operation failed.",
+         "properties": {
+            "message": {
+               "type": "string",
+               "description": "The error message.",
+               "example": "Required query parameter 'api-version' is missing."
+            },
+            "code": {
+               "type": "string",
+               "enum": [
+                  "BadRequest",
+                  "ContentFilter",
+                  "Internal",
+                  "TooManyRequests",
+                  "Unauthorized",
+                  "NotFound",
+                  "OperationNotInTerminalState",
+                  "BlockedImages",
+                  "RequestTimout"
+               ],
+               "description": "The error code."
+            },
+            "details": {
+               "type": "array",
+               "items": {
+                  "$ref": "#/definitions/InnerError"
+               }
+            }
+         },
+         "required": [
+            "message",
+            "code"
+         ]
+      },
+      "ErrorResponse": {
+         "type": "object",
+         "properties": {
+            "error": {
+               "$ref": "#/definitions/Error"
+            }
+         },
+         "required": [
+            "error"
+         ]
+      },
+      "ImageResult": {
+         "type": "object",
+         "description": "The image url if successful, and an error otherwise.",
+         "properties": {
+            "url": {
+               "type": "string",
+               "description": "The image url.",
```

Comment:
> "The image url. [](http://example.com/codeflow?start=29&length=16)

Please expand the descriptions for the important properties, like this is the result picture's url to GET.